examples/log-attack2/index.sol

```diff
@@ -0,0 +1,63 @@
+pragma abicoder v2;
+
+pragma solidity ^0.8.0;
+
+struct T1 {
+  address payable v35;
+  address payable v36;
+  uint256 v37;
+  uint256 v38;
+  }
+struct T2 {
+  address payable v36;
+  uint256 v37;
+  uint256 v38;
+  }
+struct T3 {
+  bool svs;
+  T2 msg;
+  }
+struct T5 {
+  T1 svs;
+  bool msg;
+  }
+struct T6 {
+  uint256 v84;
+  }
+struct T7 {
+  bool svs;
+  T6 msg;
+  }
+contract LogAttack2  {
+    
+  event e1(T3 _a);
+
+    constructor () payable {
+    }
+  function m1(address payable, uint256 amt) external payable {
```

Comment:
The key of THIS file is to have one function that is called by Reach's `m1` but creates a log entry for `e2`, with the goal being that Bob will see the FAKE `e2` when he goes looking for the real `e2`, because it occurs earlier in time

Reply:
i modified so .sol emits an e2 event for m1 and now bob is seeing fake e2 events(v35,v36,v37,v38) when m1 is called

Reply:
Can you update the PR so it is clear and ready to go, then we'll figure out how to fix it. I believe that right now we sort by logIndex, but need to consider only logIndex 0

Reply:
im going to keep trying  at this tonight, , i logged e2 twice in m1 and m2 because i wanted see and compare the logs